src/pages/home/report/ReportActionItemBasicMessage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import Text from '../../../components/Text';
+import styles from '../../../styles/styles';
+
+const propTypes = {
+    message: PropTypes.string.isRequired,
+    children: PropTypes.element,
+};
+
+const defaultProps = {
+    children: null,
+};
+
+function ReportActionItemBasicMessage(props) {
```

Comment:
Not a fan of this name as it doesn't seem specific enough, but I'm also having trouble coming up with a better suggestion 😅

Reply:
yeah well, it's supposed to be a generic name, so we can easily pass just a message for any kind of action.